interpolated-defaults.tf

```diff
@@ -1,2 +1,2 @@
 data "azurerm_subscription" "current" {
```

Comment:
Is this needed? I couldn't find it in use anywhere

Reply:
deleted as not required.